environments/perl/README.md

```diff
@@ -0,0 +1,14 @@
+# Fission Perl environment
+
+This environment is based on Dancer2 and Twiggy.
+
+You can use AnyEvent here.
+
+## Example
+
+```perl
+# hello.pm
+return sub {
+    return 'Hello, Perl!';
+}
```

Comment:
Could you make this a (also) a separate file. That way users can add the function easily using `fission fn create --name foo --code https://github.com/fission/fission/blob/master/environments/perl/hello.pm` to try it out.

Reply:
Added hello.pm for this, this one got more documentation.